libs/design/tree/src/README.md

```diff
@@ -0,0 +1,50 @@
+# Tree
+`DaffTreeComponent` is used to display hierarchial data.
+s
+## Usage
+```
+<daff-tree>
+  <daff-tree-item>
+	  <div daffTreeItemTitle>Design</div>
+	  <daff-tree-item>
+        <div daffTreeItemTitle>Foundations</div>
+        <a daff-tree-item [active]="rla.isActive" routerLink="/tree" routerLinkActive #rla="routerLinkActive">
+          <span daffTreeItemTitle>Color</span>
+        </a>
+        <a daff-tree-item routerLink="/tree">
+          <span daffTreeItemTitle>Typography</span>
+        </a>
+      </daff-tree-item>
+      <daff-tree-item>
+			<div daffTreeItemTitle>Components</div>
+			<daff-tree-item>
+				<span daffTreeItemTitle>Atoms</span>
+				<a daff-tree-item>
+					<span daffTreeItemTitle>Button</span>
+				</a>
+				<a daff-tree-item>
+					<span daffTreeItemTitle>Checkbox</span>
+				</a>
+			</daff-tree-item>
+		</daff-tree-item>
+	</daff-tree-item>
+	<daff-tree-item>
+		<div daffTreeItemTitle>Packages</div>
+		<daff-tree-item>
+			<div daffTreeItemTitle>@daffodil/authorizenet</div>
+			<daff-tree-item>
+				<span daffTreeItemTitle>Overview</span>
+			</daff-tree-item>
+			<a daff-tree-item>
+				<span daffTreeItemTitle>Installation</span>
+			</a>
+		</daff-tree-item>
+		<a daff-tree-item>
+			<span daffTreeItemTitle>@daffodil/cart</span>
+		</a>
+		<a daff-tree-item>
+			<span daffTreeItemTitle>@daffodil/category</span>
+		</a>
+	</daff-tree-item>
+</daff-tree>
+```
```

Comment:
These docs are insufficient. Talk about the nesting behavior of the trees and tree items.

I think the most confusing part of these examples is the `routerLinkActive` pattern. Explain how this works and the motivation behind it or replace it with something simpler.

Reply:
`routerLinkActive` is only added to force a tree-item to be active so use can see the active/selected UI. I'm not sure if there's a better way to showcase it? Or just add a note that says `routerLinkActive` is purely added to showcase the UI.

Reply:
This is reworked and no longer relevant. See the new docs in https://github.com/graycoreio/daffodil/blob/develop/libs/design/tree/README.md